script/Arcueid/Day 9/09_00_ARC09_4_1.txt

```diff
@@ -47,7 +47,7 @@ I don't think I'd even be able to make it outside without you noticing."
 [sha:d7126a112a17a5facdadab78feb39b41a9ada27c]{
 -- Page 224, Offset 21916.
 -- 「――――よかった」
-"%{i}Sigh%{/i}... Thank goodness."
+"―――Thank goodness..."
```

Comment:
A lot of these lines reached their current state after long, arduous, internal discussion that might not necessarily be reflected in comments. As such, I'd appreciate avoiding going for consistency with punctuation like this.

Please revert.